spec.yaml

```diff
@@ -195,10 +213,16 @@ paths:
                 "code": "RESOURCE_DELETION_STARTED",
                 "message": "The cluster with ID 'wqtlq' is being deleted"
```

Comment:
Not your fault probably but here is a full stop missing at the end of the sentence.